### PR TITLE
[Docs] update references and metrics post Mysticeti launch

### PR DESCRIPTION
**Pull request by mwtian**
## Description 

Checkpoint rate is now ~4/s post Mysticeti launch.

Narwhal and Bullshark are no longer running in Sui and references to them need to be migrated to Mysticeti.

## Test plan 

CI

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

**Review by arun-koshy — APPROVED** [review]

File: docs/content/guides/operator/validator-committee.mdx [hunk 1]
```diff
@@ -32,21 +32,16 @@ When a validator receives a transaction from a client, it first performs transac
 
 The process of collecting validator signatures on a transaction into a certificate and the process of submitting certificates can be performed in parallel. The client can simultaneously multicast transactions/certificates to an arbitrary number of validators. Alternatively, a client can outsource either or both of these tasks to a third-party service provider. This provider must be trusted for liveness (it can refuse to form a certificate), but not for safety (it cannot change the effects of the transaction, and does not need the user's secret key).
 
-### Certificates
+### Certificates / certified transactions
```
Comment (ronny-mysten) [inline docs/content/guides/operator/validator-committee.mdx:35]:
```suggestion
### Certificates / certified transactions {#certificates}
```

File: docs/content/guides/operator/validator-committee.mdx [hunk 2]
```diff
@@ -32,21 +32,16 @@ When a validator receives a transaction from a client, it first performs transac
...
 
-## The role of Narwhal and Bullshark
+## The role of consensus
 
-Sui takes advantage of Narwhal and Bullshark as its mempool and consensus engines. Narwhal/Bullshark (N/B) is also implemented in Sui so that when Byzantine agreement is required it uses a high-throughput DAG-based consensus to manage shared locks while execution on different shared objects is parallelized.
+Sui uses a high-throughput DAG-based consensus engine implementing the Mysticeti algorithm to sequences certified transactions. Consensus produces a sequence of commits. In each commits, there is a sequence of certified transactions.
```
Comment (ronny-mysten) [inline docs/content/guides/operator/validator-committee.mdx:43]:
```suggestion
Sui uses a high-throughput DAG-based consensus engine implementing the Mysticeti algorithm to sequences certified transactions. Consensus produces a sequence of commits. In each commit, there is a sequence of certified transactions.
```

File: docs/content/guides/operator/validator-committee.mdx [hunk 3]
```diff
@@ -32,21 +32,16 @@ When a validator receives a transaction from a client, it first performs transac
...
 
-Narwhal enables the parallel ordering of transactions into batches that are collected into concurrently proposed blocks, and Bullshark defines an algorithm for executing the DAG that these blocks form. N/B combined builds a DAG of blocks, concurrently proposed, and creates an order between those blocks as a byproduct of the building of the DAG. But that order is overlaid on top of the causal order of Sui transactions (the "payload" of Narwhal/Bullshark here), and does not substitute for it:
+The order of transactions sequenced by consensus determines the relative order they can operate on each shared object. Note that executions of transactions touching different shared objects are parallelized on multiple cores.
```
Comment (ronny-mysten) [inline docs/content/guides/operator/validator-committee.mdx:45]:
```suggestion
The order of transactions that consensus sequences determines the relative order they can operate on each shared object. Note that executions of transactions touching different shared objects are parallelized on multiple cores.
```

File: docs/content/guides/operator/validator-tasks.mdx [hunk 4]
```diff
@@ -57,10 +57,8 @@ Sui node uses the following ports by default:
 
 | protocol/port | reachability     | purpose                           |
 | ------------- | ---------------- | --------------------------------- |
-| TCP/8080      | inbound          | protocol/transaction interface    |
-| UDP/8081      | inbound/outbound | narwhal primary interface         |
-| UDP/8082      | inbound/outbound | narwhal worker interface          |
-| TCP/8083      | localhost        | sui -> narwhal interface          |
+| TCP/8080      | inbound          | validator / transaction interface |
```
Comment (ronny-mysten) [inline docs/content/guides/operator/validator-tasks.mdx:60]:
```suggestion
| TCP/8080      | inbound          | validator/transaction interface |
```

File: docs/content/references/exchange-integration-guide.mdx [hunk 5]
```diff
@@ -192,7 +192,7 @@ Sui is a DAG-based blockchain and uses checkpoints for node synchronization and
 
 - Sui creates checkpoints and adds finalized transactions. Note that transactions are finalized even before they are included in a checkpoint
 - Checkpoints do not fork, roll back, or reorganize.
-- Sui creates one checkpoint about every 3 seconds.
+ - Sui creates about 4 checkpoints every second. The most up-to-date stat can be found on the [Sui public dashboard](https://metrics.sui.io/public-dashboards/4ceb11cc210d4025b122294586961169).
```
Comment (ronny-mysten) [inline docs/content/references/exchange-integration-guide.mdx:195]:
```suggestion
- Sui creates about four checkpoints every second. Find the most up-to-date stat on the [Sui public dashboard](https://metrics.sui.io/public-dashboards/4ceb11cc210d4025b122294586961169).
```

**Review by ronny-mysten — APPROVED** [review]
Thanks @mwtian !

**Pull request merged**